File: Ejercicios Resueltos/Lenguaje C/Ejercicios Basicos/Ejercicio-26/ejercicio26.c
// Ejercicio 26
// Calcular el cambio de monedas en dólares y euros al ingresar cierta cantidad de dinero en
// pesos. 

#include <stdio.h>

void main(){
    
    float dolares, euros, pesos, cambioDolares, cambioEuros;

    printf("Ingrese el valor de compra de dólares: ");
    scanf("%f", &dolares);

    printf("Ingrese el valor de compra de euros: ");
    scanf("%f", &euros);

    printf("Ingrese la cantidad de pesos que desea cambiar: ");
    scanf("%f",&pesos);

    cambioDolares=pesos/dolares;
    cambioEuros=pesos/euros;

    printf("Pesos a cambiar: %.2f\n",pesos);
    printf("Dolares: %.2f\n",cambioDolares);
    printf("Euros: %.2f\n",cambioEuros);

    printf("\nPresione ENTER para continuar...");
    getchar();
    getchar();

}

File: Ejercicios Resueltos/Lenguaje Python/Ejercicios Basicos/Ejercicio-26/ejercicio26.py
# Ejercicio 26
# Calcular el cambio de monedas en dólares y euros al ingresar cierta cantidad de dinero en
# pesos.

print("EJERCICIO 26\n\n\n")

dolar = float(input("Ingrese el valor de dolar: "))
euro = float(input("Ingrese el valor de euro: "))
dinero = float(input("Ingreso dinero a cambiar: "))

valorDolar = dinero / dolar
valorDolar = round(valorDolar,2)
valorEuro = dinero / euro
valorEuro = round(valorEuro,2)

print(f'''
    ${dinero} -->

    dolar = ${valorDolar}
    euro = ${valorEuro}

''')

input("Presione ENTER para continuar...")